ui/server/src/main/scala/pl/touk/nussknacker/ui/uiresolving/UIProcessResolving.scala

```diff
@@ -43,7 +43,8 @@ class UIProcessResolving(validation: ProcessValidation, substitutorByProcessingT
       .getOrElse(canonical)
     val displayable = ProcessConverter.toDisplayable(substituted, processingType, businessView)
     val uiValidations = validation.uiValidation(displayable)
-    new ValidatedDisplayableProcess(displayable, uiValidations.add(validationResult).withClearedTypingInfo)
```

Comment:
Shouldn't we remove all withClearedTypingInfo for UI to work in all cases?

Reply:
done:
https://github.com/TouK/nussknacker/pull/1222/commits/ea7704364eca5bb38d774d8176ddb11693c91d7a